.htaccess

```diff
@@ -156,14 +156,61 @@
 
 	# Anything in sapphire/en/<version>/contributing is redirected to trunk, the info in there is usually not version specific,
 	# and only updated on master (e.g. contribution and coding guidelines)
-	#RewriteRule ^framework/en/contributing/?(.*) http://doc.silverstripe.org/framework/en/trunk/contributing/$1 [R=301,L]
-	#RewriteRule ^framework/en/[\d.]*/contributing/?(.*) http://doc.silverstripe.org/framework/en/trunk/contributing/$1 [R=301,L]
-
+	# RewriteRule ^framework/en/contributing/?(.*) http://doc.silverstripe.org/framework/en/trunk/contributing/$1 [R=301,L]
+	# RewriteRule ^framework/en/[\d.]*/contributing/?(.*) http://doc.silverstripe.org/framework/en/trunk/contributing/$1 [R=301,L]
+
+	RewriteRule ^changelog/?$ /en/changelogs [R=301,L]
+	RewriteRule ^installation-into-subversion/?$ /en/getting_started/composer [R=301,L]
+	RewriteRule ^modules:rssaggregator/?$ http://addons.silverstripe.org [R=301,L]
+	RewriteRule ^memberauthenticator/?$ /en/developer_guides/security/authentication [R=301,L]
+	RewriteRule ^modules:mssql/?$ http://addons.silverstripe.org/add-ons/silverstripe/mssql [R=301,L]
+	RewriteRule ^authenticator/?$ /en/developer_guides/security/authentication [R=301,L]
+	RewriteRule ^recipes:combining_files/?$ /en/developer_guides/templates/requirements [R=301,L]
+	RewriteRule ^datefield/?$ /en/developer_guides/forms/field_types/datefield [R=301,L]
+	RewriteRule ^member/?$ /en/developer_guides/security/member [R=301,L]
+	RewriteRule ^en/topics/datamodel/?$ /en/developer_guides/model/data_model_and_orm [R=301,L]
+	RewriteRule ^treedropdownfield/?$ /en/developer_guides/forms/field_types/common_subclasses [R=301,L]
+	RewriteRule ^(.*)/tutorials/1-building-a-basic-site/?$ /$1/tutorials/building-a-basic-site [R=301,L]
+	RewriteRule ^en/topics/security/?$ /en/developer_guides/security [R=301,L]
```

Comment:
It might be a lot of the top 50 get cleared with this simple improve to many existing rules.

Getting there... ;) Looking forward to getting a final PR and merge on this (hope you have learned a bunch!).